pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/PicklistQuestionFormatStrategy.java

```diff
@@ -35,15 +35,6 @@
             props.put(definition.getStableId(), MappingUtil.newTextType());
         }
 
-        for (PicklistGroupDef group : definition.getGroups()) {
```

Comment:
cleanup. group options are already included in definition.getAllPicklistOptions()